users/zeineldeen/experiments/conformer_att_2022/librispeech_960/seq_train_helpers.py

```diff
@@ -4,7 +4,7 @@
 def get_trans_lstm_lm():
     """
     Returns subnetwork of a LM trained only on transcripts. The LM architecture is close to ASR decoder architecture.
-    PPL = ?
+    PPL = 76 (train) ; 112 (dev clean+other)
```

Comment:
I took the liberty of filling out the `?` ;)

Reply:
Thanks.